ResearchKit/Common/ORKStepViewController.m

```diff
@@ -284,6 +284,18 @@ - (BOOL)hasBeenPresented {
     return _hasBeenPresented;
 }
 
+- (BOOL)canChangeStepResult {
+    if (!self.step || !self.taskViewController) {
+        return NO;
+    }
+    BOOL isReviewStepStandalone = NO;
+    ORKReviewStep *reviewStep = [self.taskViewController reviewStepForStep:self.step];
+    if (reviewStep) {
+        isReviewStepStandalone = reviewStep.isStandalone;
+    }
+    return self.step.isBeingReviewed ? !isReviewStepStandalone : YES;
```

Comment:
Is it easier to let taskVC is figure out `canChangeStepResult` and `isBeingReviewed` and assign them to the stepVC?